src/components/Dashboard/FilingHistoryList/CompletedIa.vue

```diff
@@ -3,9 +3,10 @@
     <h4>Incorporation Complete</h4>
 
     <p>{{getEntityName || 'A Numbered Benefit Company'}} has been successfully incorporated.</p>
+
     <p>Return to your Manage Businesses dashboard to access your business and file changes.</p>
 
-    <div class="to-dashboard-container">
+    <div class="to-dashboard-container text-center mt-6">
```

Comment:
Using Vuetify helpers to simplify things.
I left the classname for validation purposes.